windows-installer/construct.py

```diff
@@ -53,9 +53,9 @@ def _compile_wheels(repo_path, build_path, venv_python):
 def _prepare_build_tools(venv_path, venv_python, repo_path):
     print('Prepare build tools')
     subprocess.check_call([sys.executable, '-m', 'venv', venv_path])
+    subprocess.check_call([venv_python, os.path.join(repo_path, 'letsencrypt-auto-source', 'pieces', 'pipstrap.py')])
+    subprocess.check_call([venv_python, os.path.join(repo_path, 'tools', 'pip_install.py'), 'pynsist'])
     subprocess.check_call(['choco', 'upgrade', '-y', 'nsis'])
```

Comment:
Do you think it's worth pinning `nsis`? This post suggests it's possible: https://stackoverflow.com/questions/31255111/how-do-i-downgrade-a-chocolatey-package

Reply:
Totally! NSIS version pinning added.